### PR TITLE
add validators

### PR DESCRIPTION
**Pull request by verabehr**

### CODE REVIEW

File: scripts/setup-volume-testnet.sh [hunk 1]
```diff
@@ -62,4 +62,24 @@ init chase paloma1nty4gn8k2nrewy26fm62v03322fxgpq0hxssn6
...
+init 1	paloma1hj4lxqp05ntjl6ezu3qn9eyedp5p4fpdwu8gxj "$INIT_VALIDATION_AMOUNT"
+init 2	paloma1d3v3jh6l2r23y9kgzdrahx0ev8ez0g8qapsfxs "$INIT_VALIDATION_AMOUNT"
+init 3	paloma16jhsvx4zrukkjqd9akfawx2tzduyx4uxgtjj42 "$INIT_VALIDATION_AMOUNT"
+init 4	paloma15gvyk43x406v7kcd4rff5qfutqmcnpj3p4ea9g "$INIT_VALIDATION_AMOUNT"
+init 5	paloma1z9fgzh7mzqgu33pdkxw0dqmqgm9l8exj4nggcp "$INIT_VALIDATION_AMOUNT"
+init 6	paloma1ljg6ed0pzc3xpqtareyfp6h4fpngs7nw0nnu2j "$INIT_VALIDATION_AMOUNT"
+init 7	paloma13uslh0y22ffnndyr3x30wqd8a6peqh255hkzrz "$INIT_VALIDATION_AMOUNT"
+init 8	paloma1qf7np0rp3qutvn08vc6qz0y7ffc02cncpam3a7 "$INIT_VALIDATION_AMOUNT"
+init 9	paloma10y227j9d09pckexy32v2gckerj9a0kcepc7zsh "$INIT_VALIDATION_AMOUNT"
+init 10	paloma1swa5kcf9cl5dx2ypx0c5r9e5qdfnzp9wq59z2l "$INIT_VALIDATION_AMOUNT"
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:74]: I would

```
→9
10
```

**Review by measure-fi — APPROVED** [review]
lgtm, minor formatting nitpick

File: scripts/setup-volume-testnet.sh [hunk 2]
```diff
@@ -62,4 +62,24 @@ init chase paloma1nty4gn8k2nrewy26fm62v03322fxgpq0hxssn6
...
 
+init 1	paloma1hj4lxqp05ntjl6ezu3qn9eyedp5p4fpdwu8gxj "$INIT_VALIDATION_AMOUNT"
+init 2	paloma1d3v3jh6l2r23y9kgzdrahx0ev8ez0g8qapsfxs "$INIT_VALIDATION_AMOUNT"
+init 3	paloma16jhsvx4zrukkjqd9akfawx2tzduyx4uxgtjj42 "$INIT_VALIDATION_AMOUNT"
+init 4	paloma15gvyk43x406v7kcd4rff5qfutqmcnpj3p4ea9g "$INIT_VALIDATION_AMOUNT"
+init 5	paloma1z9fgzh7mzqgu33pdkxw0dqmqgm9l8exj4nggcp "$INIT_VALIDATION_AMOUNT"
+init 6	paloma1ljg6ed0pzc3xpqtareyfp6h4fpngs7nw0nnu2j "$INIT_VALIDATION_AMOUNT"
+init 7	paloma13uslh0y22ffnndyr3x30wqd8a6peqh255hkzrz "$INIT_VALIDATION_AMOUNT"
+init 8	paloma1qf7np0rp3qutvn08vc6qz0y7ffc02cncpam3a7 "$INIT_VALIDATION_AMOUNT"
+init 9	paloma10y227j9d09pckexy32v2gckerj9a0kcepc7zsh "$INIT_VALIDATION_AMOUNT"
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:73]:
```suggestion
init  1 paloma1hj4lxqp05ntjl6ezu3qn9eyedp5p4fpdwu8gxj "$INIT_VALIDATION_AMOUNT"
init  2 paloma1d3v3jh6l2r23y9kgzdrahx0ev8ez0g8qapsfxs "$INIT_VALIDATION_AMOUNT"
init  3 paloma16jhsvx4zrukkjqd9akfawx2tzduyx4uxgtjj42 "$INIT_VALIDATION_AMOUNT"
init  4 paloma15gvyk43x406v7kcd4rff5qfutqmcnpj3p4ea9g "$INIT_VALIDATION_AMOUNT"
init  5 paloma1z9fgzh7mzqgu33pdkxw0dqmqgm9l8exj4nggcp "$INIT_VALIDATION_AMOUNT"
init  6 paloma1ljg6ed0pzc3xpqtareyfp6h4fpngs7nw0nnu2j "$INIT_VALIDATION_AMOUNT"
init  7 paloma13uslh0y22ffnndyr3x30wqd8a6peqh255hkzrz "$INIT_VALIDATION_AMOUNT"
init  8 paloma1qf7np0rp3qutvn08vc6qz0y7ffc02cncpam3a7 "$INIT_VALIDATION_AMOUNT"
init  9 paloma10y227j9d09pckexy32v2gckerj9a0kcepc7zsh "$INIT_VALIDATION_AMOUNT"
```

**Comment by taariq** [comment]
@measure-fi is this okay to merge?

**Comment by measure-fi** [comment]
@taariq Yes, in fact it has already been used on the most recent genesis.

**Pull request merged**